src/Analyzers/Core/CodeFixes/CodeFixesResources.resx

```diff
@@ -168,4 +168,7 @@
   <data name="Convert_type_to_0" xml:space="preserve">
     <value>Convert type to '{0}'</value>
   </data>
+  <data name="Make_this_declaration_of_0_partial" xml:space="preserve">
+    <value>Make this declaration of '{0}' partial</value>
```

Comment:
I think `Add 'partial' modifier` works better here.

Reply:
Well, this one is... arguable:
1) I looked through existing code fixes resourses, and there are no patterns like `Add X modifier` there (at least on `Analizers` layer). At the same time there are multiple cases of `Make X`: `Make static`, `Make class 'abstract'`, `Make '{0}' inheritable`. I think we should be consistent with naming here and produce another `Make ...` codefix title
2) While things like `static` and `abstract` apply to the whole class/method etc. the `partial` modifier is syntactic-only and is applyed to the current declaration user is fixing. I wanted to specifically emphasize that by adding the `this declaration` title part

At the same time looking at this title with fresh brain I agree, that the `of '{0}'` part is an overkill. I would shrink the title to be just `Make this declaration partial`. It would simplify things by removing the need of getting current declaration node name, but at the same time satisfy arguments I listed above. What do you think about this option?

Reply:
I disagree.  I think 'Make partial' is perfectly fine here.

>  is syntactic-only and is applyed to the current declaration user is fixing

I don't think that's a good thing.  If there are multiple parts with problems, you'll have to fix all of them up.  We should just fix all the parts.

Reply:
Ok. I nither thought about fix all nor even planned to add it, but it seems like both you and Youssef expecting me to implement it)

Reply:
>  but it seems like both you and Youssef expect me to implement it)

there are no expectations on you :)  you can implement as much or as little as you want.  We may think that the feature needs more to be checked in, but that work can be done by you, or someone else if you would prefer that.  We're happy to help out here.